.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+* @juan131 @alvneiayu @mkmik @agarcia-oss
```

Comment:
```suggestion
* @juan131 @alvneiayu @mkmik @agarcia-oss

```

Reply:
The linter is complaining about a missing newline at the end of file

Reply:
This should be all right now?